datanode/gateway/graphql/schema.graphql

```diff
@@ -654,18 +654,18 @@ type Query {
 
 enum TransferStatus {
   "Indicates a transfer still being processed"
-  Pending
+  STATUS_PENDING
   "Indicates a transfer accepted by the Vega network"
-  Done
+  STATUS_DONE
   "Indicates a transfer rejected by the Vega network"
-  Rejected
+  STATUS_REJECTED
   """
   Indicates a transfer stopped by the Vega network
   e.g: no funds left to cover the transfer
   """
-  Stopped
+  STATUS_STOPPEDah
```

Comment:
chore (blocking): This doesn't seem right

Reply:
yes seen and updated that already when reviewing.